src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs

```diff
@@ -46,7 +46,7 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                      "Source - Source location for install. Can use special 'webpi' or 'windowsfeatures' sources. Defaults to sources.",
                      option => configuration.Sources = option.remove_surrounding_quotes())
                 .Add("l|lo|local|localonly|local-only",
-                     "LocalOnly - Only search against local machine items.",
+                     "LocalOnly - Only search against local machine items. Ignores --source.",

```

Comment:
Same here. Maybe flesh this out to say when used with `--source` then it's ignored. I think what you have is very short. Not looking for ware and peace, just somewhere in the middle.

Reply:
Updated to `Ignores --source if provided, as local packages do store installation source.`